app/assets/stylesheets/views/_brexit-landing-page.scss

```diff
@@ -44,16 +44,6 @@ $red: #E61E32;
   padding: govuk-spacing(3) govuk-spacing(2) govuk-spacing(3) 0;
 }
 
-.landing-page__section-chevron {
-  padding-right: govuk-spacing(4);
-  background-image: asset-url('chevron.svg');
```

Comment:
Does this asset need removing too?

Reply:
The asset has now been removed as well, good spot!